### DISCUSSION TITLE
Advice for Directus if is good for building a data entry with a lots fields and correlated fields

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#18860] [Feature Requests]
### Summary

I've to build a very big form wizard with a lots of fields, calculated fields and correlated. I'm finding a tool that it help me to build and manage the form. Should Directus good for this?

### Motivation

N/a

### Detailed Design

N/al 

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

N/a

### Alternatives

N/a

### Adoption Strategy

N/a

**Comment by br41nslug** [comment]
This section is intended for requesting new features for Directus :) please use [Discord](https://directus.chat/) to ask any questions
